ui/v2.5/src/components/Recommendations/Recommendations.tsx

```diff
@@ -0,0 +1,218 @@
+import * as GQL from "src/core/generated-graphql";
+import { defineMessages, useIntl } from "react-intl";
+import React from "react";
+import {
+  useFindScenes,
+  useFindMovies,
+  useFindStudios,
+  useFindGalleries,
+  useFindPerformers,
+} from "src/core/StashService";
+import { SceneCard } from "src/components/Scenes/SceneCard";
+import { StudioCard } from "src/components/Studios/StudioCard";
+import { MovieCard } from "src/components/Movies/MovieCard";
+import { PerformerCard } from "src/components/Performers/PerformerCard";
+import { GalleryCard } from "src/components/Galleries/GalleryCard";
+import { SceneQueue } from "src/models/sceneQueue";
+import { ListFilterModel } from "src/models/list-filter/filter";
+import Slider from "react-slick";
+
+const Recommendations: React.FC = () => {
+  function isTouchEnabled() {
+    return "ontouchstart" in window || navigator.maxTouchPoints > 0;
+  }
+
+  const isTouch = isTouchEnabled();
+
+  const intl = useIntl();
+  const itemsPerPage = 25;
+  const scenefilter = new ListFilterModel(GQL.FilterMode.Scenes);
+  scenefilter.sortBy = "date";
+  scenefilter.sortDirection = GQL.SortDirectionEnum.Desc;
+  scenefilter.itemsPerPage = itemsPerPage;
+  const sceneResult = useFindScenes(scenefilter);
+  const hasScenes = sceneResult.data && sceneResult.data.findScenes;
+
+  const studiofilter = new ListFilterModel(GQL.FilterMode.Studios);
+  studiofilter.sortBy = "scenes_count";
+  studiofilter.sortDirection = GQL.SortDirectionEnum.Desc;
+  studiofilter.itemsPerPage = itemsPerPage;
+  const studioResult = useFindStudios(studiofilter);
+  const hasStudios = studioResult.data && studioResult.data.findStudios;
+
+  const moviefilter = new ListFilterModel(GQL.FilterMode.Movies);
+  moviefilter.sortBy = "date";
+  moviefilter.sortDirection = GQL.SortDirectionEnum.Desc;
+  moviefilter.itemsPerPage = itemsPerPage;
+  const movieResult = useFindMovies(moviefilter);
+  const hasMovies = movieResult.data && movieResult.data.findMovies;
```

Comment:
This doesn't work for me, I didn't investigate.

Reply:
Yeah, upon testing, it's not for me either. I can push up a fix. I intended only to display the rows if content exists.